share/ecps/def2-ecp.xml

```diff
@@ -0,0 +1,133 @@
+<pseudopotential name="def2-ecp">
```

Comment:
could you put this through an xml formatter?

Reply:
Done.